pkg/controller/registry/resolver/steps.go

```diff
@@ -6,6 +6,7 @@ import (
 	"fmt"
 	"strings"
 
+	olmerrors "github.com/operator-framework/operator-lifecycle-manager/pkg/controller/errors"
```

Comment:
nit: package import should live with other OLM/API imports that start on line 19.